incubator/elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.6.5
+version: 1.6.6
```

Comment:
I would update to `1.7.0` since this is a new functionality

Reply:
Done